packages/fxa-settings/src/pages/Signin/SigninRecoveryCode/index.tsx

```diff
@@ -148,16 +158,52 @@ const SigninRecoveryCode = ({
     }
   };
 
+  const handleNavigateToRecoveryPhone = async () => {
+    clearBanners();
+    const handledError = await navigateToRecoveryPhone();
+    if (!handledError) {
+      return;
+    }
+    if (
+      handledError.errno === AuthUiErrors.BACKEND_SERVICE_FAILURE.errno ||
+      handledError.errno === AuthUiErrors.SMS_SEND_RATE_LIMIT_EXCEEDED.errno ||
+      handledError.errno === AuthUiErrors.UNEXPECTED_ERROR.errno
+    ) {
+      setBannerErrorMessage(
+        ftlMsgResolver.getMsg(
+          'signin-recovery-code-use-phone-failure',
+          'There was a problem sending a code to your recovery phone'
+        )
+      );
+      setBannerErrorDescription(
+        ftlMsgResolver.getMsg(
+          'signin-recovery-code-use-phone-failure-description',
+          'Please try again later.'
+        )
+      );
+      return;
+    }
+    setBannerErrorMessage(
+      getLocalizedErrorMessage(ftlMsgResolver, handledError)
+    );
+  };
+
   return (
     <AppLayout>
-      <FtlMsg id="signin-recovery-code-heading">
-        <HeadingPrimary>Sign in</HeadingPrimary>
-      </FtlMsg>
+      <div className="relative flex items-center mb-5">
+        <ButtonBack />
+        <FtlMsg id="signin-recovery-code-heading">
+          <HeadingPrimary marginClass="">Sign in</HeadingPrimary>
```

Comment:
nit: can remove `marginClass=""`?

Reply:
I added this because the default margin caused misalignment between the heading and back arrow